.gitignore

```diff
@@ -24,11 +24,14 @@ yarn-error.log
 include/
 .pytest_cache
 
-# Devloper Specific Files
+# Developer Specific Files
 secret_env.sh
 .autoenv
 docker.sh
+
+# Files created by make devdata
 .devdata.env
+.certificates
```

Comment:
I reckon we could change the devdata script to copy evereything it needs to be under `./devdata`.

That could be a follow up PR.

Reply:
:+1: Sounds like a good idea